posts/2021-12-15-grails-five-one.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Grails 5.1 Released
+date: Dec 15, 2021
+description: 
+author: Puneet Behl
+image: grails-blog-index-2.png
+---
+
+# [%title]
+
+[%author]
+
+[%date]
+
+The [Grails Foundation™](https://grails.org/foundation/index.html) is pleased to announce a new minor release: Grails framework 5.1!
+
+This release of the Grails framework includes a number of bug fixes and improvements for Gradle 7.2 support, plus some updates to our plugins and dependencies.
+
+The dependencies that have been updated include:
+
+- Spring Boot 2.6.1
+- Micronaut 3.2
+- GORM for Hibernate 7.2, which updates to Hibernate 5.6
+- GORM for MongoDB 7.2, which updates to underline Mongo driver 4.4
+
+Check-out the release notes from these libraries to learn about the new features that are included with those releases.
+
+A major change that accompanies Spring Boot 2.6 is that circular references are prohibited. In order to maintain backwards compatibility for Grails users, Grails framework 5.1 explicitly enables circular references by default. This will no longer be the case once Grails framework 6 is released.  
+
+## A Note About Semantic Versioning
+
+Grails framework 4 is the first version of the Framework that follows semantic versioning. The release of Grails framework 5.1 is not expected to break Grails 5 applications, and we recommend that you upgrade quickly to get the latest updates into your Grails 5 applications.
+
+## Upgrading to Grails Framework 5.1
+
+As this is a minor release, updating to Grails framework 5.1 should be pretty straightforward. Here are step-by step instructions:
+
+In your gradle.properties file,
+
+- Update property grailsVersion to 5.1.0
+- Update property gorm.version to 7.2.0
+
+In your build.gradle file
+
+- Update Gradle hibernate5 plugin to 7.2.0
+- Under dependencies, update hibernate-core to 5.6.2.Final
+
+## The Road Ahead
+
+- Spring Security Core Plugin
```

Comment:
explain what this means or link to an issue with an explanation. Spring Security Core 4.0.3 is Grails 5 compatible.